sdk/test/trace/tracer_provider_test.cc

```diff
@@ -27,3 +27,21 @@ TEST(TracerProvider, GetTracer)
   ASSERT_NE(nullptr, sdkTracer);
   ASSERT_EQ(processor, sdkTracer->GetProcessor());
 }
+
+TEST(TracerProvider, GetSampler)
+{
+  std::shared_ptr<SpanProcessor> processor(new SimpleSpanProcessor(nullptr));
+
+  // Create a TracerProvicer with a default AlwaysOnSampler.
+  TracerProvider tf(processor);
```

Comment:
Could we add a test for initializing a `TracerProvider` with a non-default sampler (`AlwaysOffSampler`)?